node/package.json

```
@@ -0,0 +1,15 @@
+{
+  "dependencies": {
+    "co-body": "^6.0.0"
+  },
+  "devDependencies": {
+    "@types/co-body": "0.0.3",
+    "@types/node": "12.x",
+    "@vtex/api": "6.36.3",
+    "typescript": "3.8.3"
+  },
+  "scripts": {
+    "lint": "tsc --noEmit --pretty && tslint -c tslint.json --fix './**/*.ts'"
```

why tslint?

---

🤷‍♂️
Just copied from cypress-example and didn't notice that, ty. Will change to:
```suggestion
    "lint": "tsc --noEmit --pretty"
```